src/app/components/HeroPool.tsx

```diff
@@ -0,0 +1,26 @@
+"use client"
+
+import Image from "next/image"
+
+export default function HeroPool () {
+  return (
+    <section className="relative w-full h-[500px] lg:h-[600px] flex items-center justify-center">
+      <Image
+        src="/pool.png"
+        alt="apartment life"
+        layout="fill"
```

Comment:
layout="fill" is deprecated. it should just be fill https://nextjs.org/docs/pages/api-reference/components/image#fill